public/app/features/explore/TraceView/createSpanLink.test.ts

```diff
@@ -215,14 +215,71 @@ describe('createSpanLinkFactory', () => {
       expect(linkDef).toBeUndefined();
     });
   });
+
+  describe('should return splunk link', () => {
```

Comment:
Tests look good. Let's add some test cases for the core functionality of formatting a query with tags, maybe a test case for 1) 1 tag, 2) multiple tags, 3) mapped tag names, 4) error handling for an empty query